### ISSUE TITLE
duplicate paths cause API viewer to go blank

### ISSUE THREAD

**Issue by JasonHenriksen** [42Crunch/vscode-openapi#18]
If you enter the same path twice, the entire API viewer goes blank and you do not get an error in the editor.

```
openapi: '3.0.2'
info:
  title: Swagger Petstore
  version: '1.0.0'
servers:
  - url: https://auth2.clearcollateral.com/
paths:

  /v1/authentications:
    post:
      tags:
        - authenticate
      summary: 
        Authenticate a user.  ( Needs to be used as the SECaaS wrapper )
      responses:
        '200':
          description: asedf
        '400':
          description: asedf
        '500':
          description: asedf

  /v1/authentications:
    post:
      tags:
        - authenticate
      summary: 
        Authenticate a user.  ( Needs to be used as the SECaaS wrapper )
      responses:
        '200':
          description: asedf
        '400':
          description: asedf
        '500':
          description: asedf
```

This will not load in the viewer.  But if you remove the second path, it will suddenly load.

This should be marked as an error in the editor, but not cause the viewer to go blank.

**Comment by ak1394** [comment]
Thanks for reporting! It indeed does not report parsing errors, I'll fix it.

**Comment by ak1394** [comment]
Editor needs to be able to parse JSON or YAML in order to display paths, etc. While duplicate paths issue is relatively benign and parser can sometimes work while encountering errors, other types of parsing errors could cause incorrect information to be displayed by the editor.

So, to err on the side of caution I'm going to keep current implementation where editor goes blank when parsing errors are encountered.

To alert user about parsing errors I've updated extension to display these in Errors panel of VS Code. The fix should be available in the upcoming 1.8.18 update. 

Please comment if you disagree with the fix.

**Comment by JasonHenriksen** [comment]
I think the more extensive error display is definitely the correct way to go.  Thank you.
